data/users/externalldap.py

```diff
@@ -262,27 +272,33 @@ def at_least_one_user_exists(self):
 
         has_pagination = not self._force_no_pagination
         with self._ldap.get_connection() as conn:
+            search_flt = self._clean_user_filter(self._ldap_user_filter)
```

Comment:
Can you add a test for this method being filtered in the LDAP test suite? https://github.com/quay/quay/blob/master/test/test_ldap.py#L632

Reply:
added unit test